source/support.tex

```diff
@@ -3378,7 +3378,7 @@
 \tcode{parmN}.}
 If the parameter
 \tcode{parmN}
-is of a reference type, or of a type that is not compatible with the
+is of a reference type, or of a type that is not the same as the
```

Comment:
Hmm, on reflection, I think "the same" is probably too strict. C allows cv qualifiers to differ, and IIRC also allows signed/unsigned mismatches. Maybe this is too large to fix editorially.